packages/components/src/navigation/group/index.js

```diff
@@ -41,10 +41,9 @@ export default function NavigationGroup( { children, className, title } ) {
 			<li className={ classes }>
 				{ title && (
 					<GroupTitleUI
-						as="h3"
```

Comment:
Not specific to this PR (which doesn't change the heading levels that are used), but I wonder what the usefulness is of using h3 here (rather than h2). The menu headings aren't nested such that we seem to need the additional heading level.

| **Sidebar menu** | **Header hierarchy** |
| - | - |
| <img width="303" alt="image" src="https://user-images.githubusercontent.com/1699996/139740165-ecb14ca9-03dc-48ae-9f55-9aa0b1afcb13.png"> | <img width="336" alt="image" src="https://user-images.githubusercontent.com/1699996/139739965-6d3377a3-68d2-49e3-87c7-4bbd261c7488.png"> |